Makefile

```diff
@@ -35,6 +35,25 @@ ifneq ($(GOOS),linux)
 	TAGS += nopkcs11
 endif
 
+.PHONY: help
```

Comment:
I believe it's enough to have this near the bottom of the file, not necessary to add noise here.